### ISSUE TITLE
scmsync packages need to be able to use --rpm-build-in-place

### ISSUE THREAD

**Issue by lnussel** [openSUSE/open-build-service#12891]
# Issue Description
I've created a github repo that contains a simple spec file

```
Name:           dummy
Version:        1
Release:        0
Group:          Development/Tools/Building
Summary:        Lorem ipsum
License:        MIT
Url:            http://www.opensuse.org/
BuildArch:      noarch

%description
...

%prep
%build

%files
%license LICENSE
%doc README.md

%changelog
```
The files are next to the spec file. A package in OBS with scmsync tag is pointing to the github repo:
```
<package name="dummy" project="home:lnussel:git">
  <title/>
  <description/>
  <scmsync>https://github.com/lnussel/dummy.git</scmsync>
</package>
```

The build fails as rpmbuild can't find the referenced README.md and LICENSE files.

See https://github.com/lnussel/dummy and https://build.opensuse.org/package/show/home:lnussel:git/dummy pointing to it.

Expected Result
============
It should just work :-)
Ie OBS would have to call `build --rpm-build-in-place` to avoid extra hacks in the spec file.

**Comment by mcepl** [comment]
Of course, it fails, you don’t have them as Source. See actually working package at https://build.opensuse.org/package/show/home:mcepl:neovim/vis

**Comment by lnussel** [comment]
Your package only works because of the tricks with %{_sourcedir} in %setup.  I know I could do that but IMO that is rather counterintuitive and not desirable for packagers to think about. Using --build-in-place is much more straight forward.
